bobtemplates/odoo/hooks.py

```diff
@@ -150,10 +153,39 @@ def post_render_model(configurator):
 #
 # addon hooks
 #
+def _is_git_exists():
+    return subprocess.call(
+        "type git",
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    ) == 0
+
+
+def _set_oca_website(configurator):
+    website = configurator.variables['copyright.website']
+    website = OCA_WEBSITE
+    if _is_git_exists():
```

Comment:
since there is a try/except pass this if is unnecessary

Reply:
Yes. I've added try/except when running tests failed on subprocess call.

Will remove this logic